test/unit/mock/profiles/dependencies-file/inheritance/inspec.yml

```diff
@@ -10,3 +10,5 @@ depends:
     path: ../profile_a
   - name: profile_b
     path: ../profile_b
+  - name: os-hardening
```

Comment:
we should rename the parent directory to `depndencies`, since we are testing more complex content here that files. we may split that up later